packages/web/src/components/comments/CommentActionBar.tsx

```diff
@@ -91,35 +137,137 @@ export const CommentActionBar = ({
     }
   }, [isMobile, onClickReply, toggleIsMobileAppDrawer])
 
+  // Confirmation Modal state
+  const confirmationModals: {
+    [k in ConfirmationAction]: CommentConfirmationModalState
+  } = useMemo(
+    () => ({
+      pin: {
+        messages: {
+          title: 'Pin this comment?',
+          body: (
+            <Text color='default'>
+              If you already pinned a comment, this will replace it
```

Comment:
Let's put all these in `messages`

Reply:
Moved everything into the common comments messages! avail for you to use as well 👍

Reply:
Update: I got some merge conflicts from Dylan and other's PRs on the messages so I re-organized the messages file (and updated mobile)